pandas/tests/extension/base/base.py

```diff
@@ -2,8 +2,18 @@
 
 
 class BaseExtensionTests:
+    @staticmethod
+    def assert_equal(left, right, **kwargs):
```

Comment:
AFAICT it's only `assert_series_equal` and `assert_frame_equal` that currently need to be overridden in specific extension array tests, so `assert_equal` and `assert_extension_array_equal` don't need to be changed at this time.